app/frontend/src/components/forms/TeamManagement.vue

```diff
@@ -1,31 +1,82 @@
 <template>
-  <v-card outlined>
-    <v-skeleton-loader
-      :boilerplate="false"
-      class="mx-auto"
-      :loading="loading"
-      type="card-heading, list-item-three-line@4"
-      tile
+  <div>
+    <!-- search input -->
+    <div class="team-search mt-6 mt-sm-0">
+      <v-text-field
+        v-model="search"
+        append-icon="mdi-magnify"
+        label="Search"
+        single-line
+        hide-details
+        class="pb-5"
+      />
+    </div>
+
+    <!-- team table -->
+    <v-data-table
+      class="team-table"
+      :headers="headers"
+      :items="tableData"
+      :loading="loading || updating"
+      loading-text="Loading... Please wait"
+      no-data-text="Failed to load team role data"
+      :search="search"
     >
-      <v-list-item three-line>
-        <v-list-item-content>
-          <div class="overline mb-4">TEAM MANGEMENT</div>
-          <div v-for="role in roleList" :key="role.code">
-            <div v-if="role.active">
-              <v-list-item-title class="headline my-2">{{ role.display }}</v-list-item-title>
-              <ul v-for="user in formUsers" :key="user.userId">
-                <li v-if="user.roles.includes(role.code)">{{ user.fullName }} ({{ user.username }})</li>
-              </ul>
+      <template v-slot:item="{ item, isMobile, headers }">
+        <tr v-if="isMobile" class="v-data-table__mobile-table-row">
+          <td
+            v-for="header in headers"
+            :key="header.value"
+            class="v-data-table__mobile-row"
+          >
+            <div class="v-data-table__mobile-row__header">
+              {{ header.text }}
+            </div>
+            <div class="v-data-table__mobile-row__cell">
+              <v-checkbox
+                v-if="typeof item[header.value] === 'boolean'"
+                v-model="item[header.value]"
+                @click="edited = true"
+              />
+              <div v-else>{{ item[header.value] }}</div>
             </div>
-          </div>
-        </v-list-item-content>
-      </v-list-item>
-    </v-skeleton-loader>
-  </v-card>
+          </td>
+        </tr>
+        <tr v-else>
+          <td v-for="header in headers" :key="header.value" class="text-start">
```

Comment:
I think you should add another class here for the actual role columns, enforce a standard sizing on those columns.  It looks pretty weird right now.

Reply:
maybe this will give you unique classes for each column:
class=" `${header.value}` text-start"
if that doesnt work maybe wrap the actual text in another span.
it might be solved using >>> deep css / inheritance stuff

Reply:
I tried a few things like setting the class to text-center - that had no effect.
Setting the header align definition to center actually made things worse somehow and made the checkboxes not even align under the header at all.
Matt said to leave this for now and revisit at a later date once more things have settled down.